hack/deploy-setup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# This script inits an elasticsearch-operator
+# to deploy an Elasticsearch cluster.  It assumes it is capable of login as a
+# user who has the cluster-admin role
+
+set -e
+
+source "$(dirname $0)/common"
+
+load_manifest() {
+  local repo=$1
+  local namespace=${2:-}
+  if [ -n "${namespace}" ] ; then
+    namespace="-n ${namespace}"
+  fi
+
+  pushd ${repo}/manifests
+    for m in $(ls); do
+      if [ "$(echo ${EXCLUSIONS[@]} | grep -o ${m} | wc -w)" == "0" ] ; then
```

Comment:
I guess this works - but usually if you use the old style bourne shell syntax `if [` you have to use `=` and if you use the new style bash `if [[` you have to use `==`